specification/synapse/data-plane/readme.md

```diff
@@ -76,6 +76,7 @@ input-file:
   - Microsoft.Synapse/stable/2020-12-01/sqlScripts.json
   - Microsoft.Synapse/stable/2020-12-01/triggers.json
   - Microsoft.Synapse/stable/2020-12-01/workspace.json
+  - Microsoft.Synapse/stable/2020-12-01/credential.json
```

Comment:
could you also add this to the latest tag: package-artifacts-composite-v7, we will use that tag to generate sdk.